config/locales/en.yml

```diff
@@ -460,12 +460,14 @@ en:
           success: User key has been updated.
 
       authentication_providers:
+        edit:
+          automatically_approve_accounts_hint_html: "Even if the <a href=%{link}>account approval settings</a> say otherwise"
```

Comment:
This is to be used from **Developer portal > Settings > SSO Integrations**, right?

I can't open that screen. It raises the error:

```
Translation missing: en.provider.admin.authentication_providers.form.automatically_approve_accounts_hint_html
```

Reply:
You're correct!